lib/tasks/backend.rake

```diff
@@ -1,6 +1,7 @@
+# rubocop:disable Metrics/BlockLength
```

Comment:
Can we disable this for all rake tasks in `.rubocop.yml`?

Reply:
Would that need to be updated in govuk-lint or does it pick up local `.rubocop.yml`s ?

Reply:
Having a local overrride works. 

Might be worth adding it to govuk-lint too (like https://github.com/alphagov/govuk-lint/pull/87), but don't wait for it to get merged.